src/diffusers/loaders/lora_conversion_utils.py

```diff
@@ -2201,29 +2205,44 @@ def convert_key(key: str) -> str:
     all_keys = list(state_dict.keys())
     down_key = ".lora_down.weight"
     up_key = ".lora_up.weight"
+    a_key = ".lora_A.weight"
+    b_key = ".lora_B.weight"
 
-    def get_alpha_scales(down_weight, alpha_key):
-        rank = down_weight.shape[0]
-        alpha = state_dict.pop(alpha_key).item()
-        scale = alpha / rank  # LoRA is scaled by 'alpha / rank' in forward pass, so we need to scale it back here
-        scale_down = scale
-        scale_up = 1.0
-        while scale_down * 2 < scale_up:
-            scale_down *= 2
-            scale_up /= 2
-        return scale_down, scale_up
+    has_lora_arrow = any(down_key in k or up_key in k for k in all_keys)
+    has_lora_letter = any(a_key in k or b_key in k for k in all_keys)
```

Comment:
I would maybe change to:

`has_non_diffusers_lora_id` and `has_diffusers_lora_id`. 

Easier for readability.